collect_app/src/main/java/org/odk/collect/android/audio/AudioRecordingControllerFragment.java

```diff
@@ -50,6 +52,12 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         viewModel.getCurrentSession().observe(getViewLifecycleOwner(), session -> {
             if (session == null) {
                 binding.getRoot().setVisibility(GONE);
+            } else if (session.getFailedToStart()) {
+                binding.getRoot().setVisibility(GONE);
+                new MaterialAlertDialogBuilder(requireContext())
```

Comment:
Hmmm I think this might cause problems on rotation as a second dialog would get opened. Not a hude deal but should probably fix.